include/triton/Dialect/Triton/IR/TritonAttrDefs.td

```diff
@@ -87,5 +87,17 @@ def TT_ProgramDim : I32EnumAttr<
     let cppNamespace = "::mlir::triton";
 }
 
+// Rounding mode.
+def TT_RoundingModeAttr : I32EnumAttr<
+    "RoundingMode", "",
+    [
+        I32EnumAttrCase<"RTZ", 0, "rtz">,
+        I32EnumAttrCase<"RTNE", 1, "rtne">,
+        I32EnumAttrCase<"RU", 2, "ru">,
```

Comment:
I think I'll get rid of the ru and rd, I see no point in having them here since they ended up not being used.

Reply:
Sounds reasonable to me, but if we don't support all feasible rounding modes -- and if for the modes we support we don't support all combinations of dtypes -- then to me it seems important that we document what we *do* support, and that we provide good error messages.